packages/angular/src/migrations/update-11-0-0/update-builders-config.spec.ts

```diff
@@ -72,25 +72,32 @@ describe('11.0.0 Migration: Update Builders Config', () => {
 
     const workspace = await getWorkspace(result);
 
-    expect(workspace.projects.get('app1').targets.get('build').options).toEqual(
-      {
-        main: 'main.ts',
-        scripts: [
-          'scripts.ts',
-          {
-            input: 'more-scripts.ts',
-            inject: false,
-          },
-        ],
-        styles: [
-          'styles.scss',
-          {
-            input: 'more-styles.scss',
-            inject: false,
-          },
-        ],
-      }
-    );
+    const proxyOptions = workspace.projects
+      .get('app1')
+      .targets.get('build').options;
+    const options = {
```

Comment:
This looks fine but do you know why this is necessary?

Reply:
@Coly010 was looking in more depth into it. 

The UnitTestTree we use in the test, leaves `workspace.json` in the invalid state (has a trailing comma after targets/build/options/scripts) after migration. The migration itself works fine.

This leads to missing build block on `getWorkspace` which then fails the test.

I will check why the invalid JSON happens in a followup PR, as there might be a hidden issue waiting to blow up.